examples/with-route-as-modal-page/README.md

```diff
@@ -0,0 +1,48 @@
+# with-route-as-modal-page
```

Comment:
@lfades  I had deleted this directory and used rebasing + force push to update the branch. Seems like your local might have been out of sync when you just pushed.. 

Just wanted to call it out here

Reply:
Oh...

Reply:
Should be okay now 😌